### ISSUE TITLE
Can't disable chaperone

### ISSUE THREAD

**Issue by HugoZink** [OpenVR-Advanced-Settings/OpenVR-AdvancedSettings#257]
I can't disable the SteamVR chaperone. The visibility won't go below 30% in the chaperone settings. Ticking "disable chaperone" does not actually do anything.

I'm using an Oculus Rift. In SteamVR room setup I chose for a small standing playspace (no room-scale), because SteamVR room setup won't let me trace my room with the controllers.

I have Oculus' own guardian system disabled entirely. No matter what I do, I still get the faint blue circle on the floor. No amount of setting changes will get rid of it.

**Comment by HugoZink** [comment]
I finally managed to set up "room-scale" by cheating with my sensors a little bit, and the chaperone can now be properly disabled.

It seems like the standing "Chaperone" circle can no longer be disabled when using a Standing only playspace in later OpenVR advanced settings versions. This seems like a bug.

**Comment by ykeara** [comment]
unfortunately we are mostly limited by OpenVR api here.

1. chaperone Opacity is intentionally limited to 30% (was changed about a year ago?) on the OpenVR end.
2. Standing only does not have the same api behavior as room-scale so our work-around does not work to hide the chaperone in this configuration.


tl;dr while not ideal behavior its really best given the situation, and is expected, as it is really outside of our control.

Closing issue.

**Comment by HugoZink** [comment]
That's odd, when was this changed? I was on an older version of OpenVR Advanced Settings and I had the chaperone disabled all the time. I updated after a fresh Windows install and only then did this issue pop up.

**Comment by ykeara** [comment]
to be fair if you used a really old OVR version you could  still get this  behavior, but with many of the features it would be incompatible with current state now.

in terms of fade distance, its just the api.... again the call doesn't do anything if not in roomscale. and we don't actively check because its somewhat pointless to do so.

It might be possible to force people into "room-scale" but again I would just suggest current guidance is to set-up roomscale. even if you have to trace over areas that you can't really occupy.... i.e. above desk etc.
